build.xml

```diff
@@ -182,52 +182,44 @@
     <!-- Create demo users. -->
     <target name="create-demo-users" description="Create demo users.">
         <foreach list="${drupal.demo.users}" param="drupal.demo.user" target="create-demo-user" delimiter="," />
-        <!-- Create a user with only "authenticated user" role assigned. -->
+        <foreach list="${drupal.demo.authenticated_users}" param="drupal.demo.user" target="create-demo-authenticated_user" delimiter="," />
+    </target>
+
+    <!-- Create a demo user. Subtarget of "create-demo-users". -->
+    <target name="create-demo-user">
```

Comment:
Ah, ok I kinda proved myself wrong :D I did initially implement this like this and then thought that it would be wrong. But I still kept these changes in case we would need more than one user. I will revert it. :)